Lib/idlelib/editor.py

```diff
@@ -803,6 +807,16 @@ def colorize_syntax_error(self, text, pos):
             text.mark_set("insert", pos + "+1c")
         text.see(pos)
 
+    def UpdateCursorBlink(self):
+        "Update the cursor blink configuration."
+        cursorblink = idleConf.GetOption(
+                'main', 'EditorWindow', 'cursor-blink', type='bool')
+        if not cursorblink:
+            self.text.config(insertofftime=0)
+        else:
+            # Restore the original value
+            self.text.config(insertofftime=self.text._insertofftime)
```

Comment:
```suggestion
            self.text['insertofftime'] = self.blink_off_time
```
I prefer the pythonic option getting/setting.